app/main/views/buyers.py

```diff
@@ -12,26 +12,21 @@
 @role_required('admin', 'admin-ccs-category')
 def find_buyer_by_brief_id():
     brief_id = request.args.get('brief_id')
+    brief = None
+    status_code = 200
 
     try:
-        brief = data_api_client.get_brief(brief_id).get('briefs')
-
+        brief = data_api_client.get_brief(brief_id)
     except:
-        flash('no_brief', 'error')
-        return render_template(
-            "view_buyers.html",
-            users=list(),
-            brief_id=brief_id
-        ), 404
+        if not brief and 'brief_id' in request.args:
```

Comment:
Because the brief assignment will never fully execute if we are at this point (ie if there is an exception in getting and assigning the brief) brief will always be None here so we don't need to check `if not brief`

Reply:
Had spotted this too but didn't fix.  Will do.